modules/change-collection-method-ebpf.adoc

```diff
@@ -3,17 +3,17 @@
 // * upgrading/upgrade-operator.adoc
 :_mod-docs-content-type: CONCEPT
 [id="change-collection-method-ebpf_{context}"]
-= Changing the collection method to EBPF
+= Changing the collection method to CORE_BPF
 
 [role="_abstract"]
-If the cluster that you are upgrading contains the `SecuredCluster` CR, you must ensure that the per node collection setting is set to EBPF before you upgrade.
+If the cluster that you are upgrading contains the `SecuredCluster` CR, you must ensure that the per node collection setting is set to CORE_BPF before you upgrade.
```

Comment:
```suggestion
If the cluster that you are upgrading contains the `SecuredCluster` CR, you must ensure that the per node collection setting is set to CORE_BPF before you upgrade, if you are upgrading from 4.1 or later. Otherwise set the collection method to EBPF. If you want the collection method to be EBPF after the upgrade, you will need to set the forceCollection option to true and make sure that the collection method is EBPF.
```